gamestonk_terminal/cryptocurrency/ta_crypto_controller.py

```diff
@@ -0,0 +1,98 @@
+"""Technical Analysis Controller Module"""
+__docformat__ = "numpy"
+
+from datetime import datetime
+import pandas as pd
+import matplotlib.pyplot as plt
+from prompt_toolkit.completion import NestedCompleter
+
+from gamestonk_terminal import feature_flags as gtff
+from gamestonk_terminal.helper_funcs import get_flair
+from gamestonk_terminal.menu import session
+from gamestonk_terminal.technical_analysis.ta_controller import (
+    TechnicalAnalysisController as TAC,
+)
+
+
+class CryptoTechnicalAnalysisController(TAC):
```

Comment:
Why did we add a TA specifically for crypto? Since it uses the same functionalities, I'm sure we're able to somehow manage it from the TA perspective of things. That would be ideal, as it would remove duplicate code.